bin/oneoff/wipe_poste_deliveries_and_contacts_emails

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env ruby
+
+# This script removes plaintext emails from the contacts and psote_deliveries
+# tables for all and young Code Studio students, respectively.
+
+require_relative '../../deployment'
+require_relative '../../lib/cdo/db'
+
+# Populate a hash of hashed_emails to birthdays.
+puts 'Populating hashed_email_to_birthday...'
+hashed_email_to_birthday = {}
+DASHBOARD_DB[:users].
+  where(user_type: 'student').
+  where('hashed_email IS NOT NULL').
+  paged_each(rows_per_fetch: 10_000) do |user|
+  hashed_email_to_birthday[user[:hashed_email]] = user[:birthday].strftime('%F')
+end
+
+# Wipe the plaintext email of young students.
+puts 'Wiping contacts...'
+PEGASUS_DB[:contacts].each do |contact|
+  birthday = hashed_email_to_birthday[contact[:hashed_email]]
+  if birthday && birthday > '2003-07-01'
+    PEGASUS_DB[:contacts].where(id: contact[:id]).update(email: '')
```

Comment:
At least locally, it seems as if `paged_each` (on line 21) cannot be used with this update. Recommendations on how batching should be done, given this?